### ISSUE TITLE
Please build dovecot with gssapi support going forward

### ISSUE THREAD

**Issue by nshalman** [joyent/pkgbuild#2]
This change fixes the build for the 2015Q1 branch, but obviously I'd really like this change to apply to all future branches as well.
```diff
diff --git a/include/pkgoptions/2015Q1.mk b/include/pkgoptions/2015Q1.mk
index d3d1828..1524e77 100644
--- a/include/pkgoptions/2015Q1.mk
+++ b/include/pkgoptions/2015Q1.mk
@@ -44,7 +44,7 @@ PKG_DEFAULT_OPTIONS+=         -ruby-build-ri-db
 #    which may conflict with user preference
 #
 PKG_OPTIONS.django=            pgsql sqlite
-PKG_OPTIONS.dovecot=           sqlite
+PKG_OPTIONS.dovecot=           sqlite gssapi
 PKG_OPTIONS.postfix=           sqlite
 PKG_OPTIONS.sphinx-search=     mysql pgsql
 PKG_OPTIONS.sqlrelay=          mysql sqlite

```

**Comment by jperkin** [comment]
Done for trunk now so will be in 2015Q2, thanks!

**Comment by nshalman** [comment]
Closed in 9819835239e8b4657cd2d755bfb02e8196f48893 (at https://github.com/joyent/pkgbuild/commit/9819835239e8b4657cd2d755bfb02e8196f48893#diff-7b2542e223928a124922a38dfef3e4a9R59 )
